arwen/host/execution_builtin_test.go

```diff
@@ -85,31 +84,6 @@ func TestExecution_AsyncCall_BuiltinFails(t *testing.T) {
 	require.Equal(t, [][]byte{[]byte("hello"), {4}}, vmOutput.ReturnData)
 }
 
-func TestExecution_AsyncCall_CallBackFailsBeforeExecution(t *testing.T) {
```

Comment:
Yes, this test was deleted because a callback cannot fail with insufficient gas anymore before it even starts executing. Dynamic gas locking prevents this - although the callback can fail after it starts, of course.